### ISSUE TITLE
Form inputs should have labels

### ISSUE THREAD

**Issue by finnhodgkin** [facn2/Alpha.HAML#70]
All form inputs should have attached labels for improved accessibility.

https://github.com/facn2/Alpha.HAML/blob/master/public/index.html#L19-L26
